api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
+
+def startdb()
+  c = Common.new
```

Comment:
I think it is! I think I had decided that `c` probably stood for `container` -- something Docker-specific. A name that lets me know "these are some library functions that David / we wrote" is helpful.

Is there state in a "common"? Looking at https://github.com/dmohs/project-management/blob/master/common.rb I'm reading it as similar to a Java static file with functions -- essentially a namespace. Though `@@commands` is stateful? It's library functions + a stateful argument parser? Is it possible to reference stateless functions outside of an instance?